jsapp/js/account/accountSettingsRoute.tsx

```diff
@@ -60,36 +67,62 @@ const AccountSettings = observer(() => {
   }, []);
   useEffect(() => {
     const currentAccount = sessionStore.currentAccount;
+
     if (
-      !sessionStore.isPending &&
-      sessionStore.isInitialLoadComplete &&
-      'email' in currentAccount
+      sessionStore.isPending ||
+      !sessionStore.isInitialLoadComplete ||
+      !('email' in currentAccount) ||
```

Comment:
Since we're touching this section, I have to ask: Do we need these session store readiness checks? My impression is we could just use `'email' in currentAccount` (which I suspect is our way of making sure the store has a real user and not an anonymous user stub (extra credit if you find a better way to do that, too).